packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -76,6 +88,26 @@ const Trade = observer(() => {
         }
     }, []);
 
+    React.useEffect(() => {
+        if (!default_stake || prev_contract_type === contract_type) return;
+
+        // Set stake to default value (from contracts_for API) and disabling TP when user switched to another trade type
+        setV2ParamsInitialValues({ value: default_stake, name: 'stake' });
+        onChangeMultiple({
```

Comment:
Question: What does onChangeMultiple do?

Reply:
It's updates _trade-store_ values (call `processNewValuesAsync` function inside) and those values get to proposal request (call `requestProposal` function).

Reply:
So this is run when the trade parameters change?

Reply:
Yes. There are 2 options when we want to change trade params: onChangeMultiple and onChange. Both will do what I've described above, but onChangeMultiple allows to change couple of trad params at the same time, while onChange - only one.